src/components/Components/ImportDropdown.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useRef } from 'react'
+import { PropTypes } from 'prop-types'
+
+import { useClient } from 'cozy-client'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import ActionMenu, { ActionMenuItem } from 'cozy-ui/transpiled/react/ActionMenu'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import Button from 'cozy-ui/transpiled/react/Button'
+import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
+
+import withModal from '../HOCs/withModal'
+import ContactImportationModal from '../ContactImportationModal'
+import { getFilteredStoreUrl } from '../../helpers/store'
+
+const ImportDropdown = ({ showModal, hideModal }) => {
+  const { t } = useI18n()
+  const client = useClient()
+  const anchorRef = useRef()
+  const [menuDisplayed, setMenuDisplayed] = useState(false)
+  const showMenu = () => setMenuDisplayed(true)
+  const hideMenu = () => setMenuDisplayed(false)
+
+  const showContactImportationModal = () => {
+    showModal(<ContactImportationModal closeAction={hideModal} />)
```

Comment:
This is a bit dangerous to do that since the modal is no longer in the React tree. It will not-render in case of change of props etc... It should not be important in this particular case since only `hideModal` is passed and it should not changed.

Reply:
Okay, I'll keep that in mind. For the modals, the whole app is made like this, maybe this will be the opportunity to revisit it by migrating to `<Dialog>`.